src/app/web3charts/Abi.ts

```diff
@@ -0,0 +1,100 @@
+import Web3 from 'web3';
```

Comment:
I guess this class can be removed
if you need to store some static variables - create a file in `static` folder